Networking/Networking/Model/StripeAccount.swift

```diff
@@ -0,0 +1,128 @@
+/// Represent a Stripe Account Entity.
+///
+public struct StripeAccount: Decodable {
+    public static let gatewayID = "woocommerce-stripe"
+
+    public let status: WCPayAccountStatusEnum
+    /// Indicates whether the payment gateway is a live account that can accept actual payments, or just a test/developer account.
+    /// Not to be confused with "test mode" which is a separate concept (see `isInTestMode`)
+    /// Note: Using WCPayAccountStatusEnum is somewhat inappropriate here, since it ties this Stripe account concept
+    /// to WooCommerce Payments - although in reality both extensions use the same account on the backend.
+    /// TODO: Introduce a layer of abstraction and translate each extension's account status into a
+    /// PaymentGatewayAccountStatusEnum or somesuch.
+    ///
+    public let isLiveAccount: Bool
+    /// Indicates whether the payment gateway is currently in "Test" or "Debug" mode
+    ///
+    public let isInTestMode: Bool
+    public let hasPendingRequirements: Bool
+    public let hasOverdueRequirements: Bool
+    public let currentDeadline: Date?
+    /// An alphanumeric string set by the merchant, e.g. `MYSTORE.COM`
+    /// See https://stripe.com/docs/statement-descriptors
+    public let statementDescriptor: String
+    /// A three character lowercase currency code, e.g. `usd`
+    /// See https://stripe.com/docs/api/accounts/object#account_object-default_currency
+    public let defaultCurrency: String
+    public let supportedCurrencies: [String]
+    /// A two character country code, e.g. `US`
+    /// See https://stripe.com/docs/api/accounts/object#account_object-country
+    public let country: String
+    /// A boolean flag indicating if this Account is eligible for card present payments
+    public let isCardPresentEligible: Bool
```

Comment:
nit: maybe it's worth mentioning the full "card present payment"?
```suggestion
    public let isCardPresentPaymentEligible: Bool
```

Reply:
We'll be removing that concept (is...Eligible) from the codebase at some point soon :) We used to use it for beta gating and don't need it any more.